osfclient/cli.py

```diff
@@ -65,6 +69,39 @@ def _setup_osf(args):
 
     return OSF(username=username, password=password)
 
+def init(args):
+    """Initialize or edit an existing .osfcli.config file
+
+    """
+    # reading existing config file, convert to configparser object
+    config = config_from_file()
+    config_ = configparser.ConfigParser()
```

Comment:
you could go for
```python
config = configparser.ConfigParser()
config.read_dict({'osf': config_from_file())
```
to setup the parser with the current values of the config file.

What was your thinking behind not taking values from the environment as well?